docs/deprecated_wiki/Installing-Loris.md

```diff
@@ -32,4 +32,4 @@ Mac OS X is no longer supported as of LORIS 15.10. You may do the following for
 
 **[[NEXT: (3) Behavioural Setup|Behavioural-Database]]**
 
-**[[NEXT: (4) Imaging Setup|Imaging-Database]]**
\ No newline at end of file
+**[[NEXT: (4) Imaging Setup|Imaging-Database]]**
```

Comment:
This is causing a rebase issue - since this is a deprecated file, I would suggest undoing your change to this file instead of rebasing.